src/create-job.ts

```diff
@@ -124,4 +134,14 @@ export default async function createJob(description: JobDescription, pipelineDat
   logger.info('Finished analysis!');
 }
 
-
+async function uploadReferenceIfNeeded(description: JobDescription, pipeline: Pipeline) {
+  if (pipeline instanceof AWSPipeline) {
+    return await pipeline.uploadIfNeeded(
+      description.reference,
+      pipeline.configuration.inputBucket,
+      description.name,
+      'reference.mp4'
+    )
+  }
+  return description.reference;
+}
```

Comment:
Going through the commits, I cannot see anything that's changed since I merged main into this branch. If anything I guess something has changed due to me doing a format on the file

Reply:
Oh, well. No worries.